app/api/routes.py

```diff
@@ -1,16 +1,17 @@
 from flask import jsonify, request, current_app, url_for
-from . import api
 from ..models import Plant, Garden, GardenPlant
+from datetime import timedelta, datetime
 from flask_sqlalchemy import SQLAlchemy
 from app import db
+from . import api
 import code
-from datetime import datetime
-from datetime import timedelta
-
+# Base endpoint test used when testing deployment.
+# Also used later as a motivational message.
 @api.route('/')
 def endpoint():
     return "I need to go take a shower so I can't tell if I'm crying or not."
 
+# Get All Plants
```

Comment:
With routes looking all weird (Miss you Rails!) adding these comments really improve readability.